tests/Behat/Resources/suites.yml

```diff
@@ -116,3 +116,16 @@ default:
 
             filters:
                 tags: "@cleaning_queue && @cli"
+
+        enqueuing_products_from_route:
+            contexts:
+                - sylius.behat.context.hook.doctrine_orm
+
+                - sylius.behat.context.setup.admin_security
+                - webgriffe_sylius_akeneo.behat.context.setup.product
+                - webgriffe_sylius_akeneo.behat.context.setup.queue
+
+                - webgriffe_sylius_akeneo.behat.context.ui.admin.managing_products
+                - webgriffe_sylius_akeneo.behat.context.ui.admin.managing_queue_items
+            filters:
+                tags: "@enqueuing_products_from_route && @ui"
```

Comment:
Given the refactor mentioned above, this suite should be renamed to `ui_enqueueing_products` with filter tags `@enqueuing_products && @ui`